oioioi/contests/templates/contests/contest_rules.html

```diff
@@ -0,0 +1,54 @@
+{% extends "base-with-menu.html" %}
+{% load i18n %}
+{% load pagination_tags %}
+
+{% block title %}{% trans "Rules" %}{% endblock %}
+
+{% block main-content %}
+<h1>{% trans "Rules" %}</h1>
+<ul class="list-group list-group-light">
+    {% if scoring_type %}
+        {% blocktrans %}
+            <li class="list-group-item">The scoring and contest type is "{{ scoring_type }}".</li>
```

Comment:
HTML blocks used inside `blocktranslate` are omitted? If so, that's fine.

Reply:
My bad, the HTML blocks aren't omitted - thank you for pointing this one out!